addons/binding/pom.xml

```diff
@@ -1,111 +1,112 @@
-		<module>org.openhab.binding.rme</module>
-		<module>org.openhab.binding.rfxcom</module>
-		<module>org.openhab.binding.rfxcom.test</module>
-		<module>org.openhab.binding.rotelra1x</module>
-		<module>org.openhab.binding.russound</module>
-		<module>org.openhab.binding.samsungtv</module>
+    <module>org.openhab.binding.pioneeravr</module>
+    <module>org.openhab.binding.plugwise</module>
+    <module>org.openhab.binding.pulseaudio</module>
+    <module>org.openhab.binding.regoheatpump</module>
+    <module>org.openhab.binding.rme</module>
+    <module>org.openhab.binding.rfxcom</module>
+    <module>org.openhab.binding.rfxcom.test</module>
+    <module>org.openhab.binding.rotelra1x</module>
+    <module>org.openhab.binding.russound</module>
+    <module>org.openhab.binding.samsungtv</module>
 		<module>org.openhab.binding.seneye</module>
-		<module>org.openhab.binding.sensebox</module>
-		<module>org.openhab.binding.silvercrestwifisocket</module>
-		<module>org.openhab.binding.sleepiq</module>
-		<module>org.openhab.binding.smaenergymeter</module>
-		<module>org.openhab.binding.squeezebox</module>
-		<module>org.openhab.binding.synopanalyzer</module>
-		<module>org.openhab.binding.systeminfo</module>
-		<module>org.openhab.binding.systeminfo.test</module>
-		<module>org.openhab.binding.tankerkoenig</module>
-		<module>org.openhab.binding.tellstick</module>
-		<module>org.openhab.binding.tesla</module>
-		<module>org.openhab.binding.toon</module>
-		<module>org.openhab.binding.tplinksmarthome</module>
-		<module>org.openhab.binding.tplinksmarthome.test</module>
-		<module>org.openhab.binding.urtsi</module>
-		<module>org.openhab.binding.vitotronic</module>
-		<module>org.openhab.binding.wifiled</module>
-		<module>org.openhab.binding.windcentrale</module>
-		<module>org.openhab.binding.yamahareceiver</module>
-		<module>org.openhab.binding.zoneminder</module>
-		<module>org.openhab.binding.zway</module>
-	</modules>
+    <module>org.openhab.binding.sensebox</module>
+    <module>org.openhab.binding.silvercrestwifisocket</module>
+    <module>org.openhab.binding.sleepiq</module>
+    <module>org.openhab.binding.smaenergymeter</module>
+    <module>org.openhab.binding.squeezebox</module>
+    <module>org.openhab.binding.synopanalyzer</module>
+    <module>org.openhab.binding.systeminfo</module>
+    <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tankerkoenig</module>
+    <module>org.openhab.binding.tellstick</module>
+    <module>org.openhab.binding.tesla</module>
+    <module>org.openhab.binding.toon</module>
+    <module>org.openhab.binding.tplinksmarthome</module>
+    <module>org.openhab.binding.tplinksmarthome.test</module>
+    <module>org.openhab.binding.urtsi</module>
+    <module>org.openhab.binding.vitotronic</module>
+    <module>org.openhab.binding.wifiled</module>
+    <module>org.openhab.binding.windcentrale</module>
+    <module>org.openhab.binding.yeelight</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).